tests/dummy/app/controllers/logging-example/application-log.js

```diff
@@ -0,0 +1,375 @@
+jhghhjlk
+
+     */
+    errorAction() {
+      if (this.logLevel < 1) {
+        this.showConfirmModalDialog('Error');
+        return;
+      }
+
+      this.makeError();
+    },
+
+    /**
+     j hghh*jlk
+
+     */
+    throwAction() {
+      if (this.logLevel < 1) {
+        this.showConfirmModalDialog('Throw');
+        return;
+      }
+
+      this.makeThrow();
+    },
+
+    /**
+     * jhghhjlk
+     *
+     */
+    deprecationAction() {
+      if (this.logLevel < 6) {
+        this.showConfirmModalDialog('Deprecation');
+        return;
+      }
+
+      this.makeDeprecation();
+    },
+
+    /**
+     * jhghhjlk
+     *
+     */
+    debugAction() {
+      if (this.logLevel < 5) {
+        this.showConfirmModalDialog('Debug');
+        return;
+      }
+
+      this.makeDebug();
+    },
+
+    /**
+jhghhjlk
+
+  */
+    infoAction() {
+      if (this.logLevel < 4) {
+        this.showConfirmModalDialog('Info');
+        return;
+      }
+
+      this.makeInfo();
+    },
+    /**
+jhghhjlk
+
+     */
+    logAction() {
+      if (this.logLevel < 3) {
+        this.showConfirmModalDialog('Log');
+        return;
+      }
+
+      this.makeLog();
+    },
+
+    /**
+     * jhghhjlk
+     *
+     */
+    warnAction() {
+      if (this.logLevel < 2) {
+        this.showConfirmModalDialog('Warn');
+        return;
+      }
+
+      this.makeWarn();
+    },
+
+    confirmed(category) {
+      switch (category) {
+        case 'Assert':
+          this.makeAssert();
+          break;
+        case 'Error':
+          this.makeError();
+          break;
+        case 'Throw':
+          this.makeThrow();
+          break;
+        case 'Warn':
+          this.makeWarn();
+          break;
+        case 'Deprecation':
+          this.makeDeprecation();
+          break;
+        case 'Debug':
+          this.makeDebug();
+          break;
+        case 'Info':
+          this.makeInfo();
+          break;
+        case 'Log':
+          this.makeLog();
+          break;
+        case 'Warn':
+          this.makeWarn();
+          break;
+      }
+    },
+
+    /**
+     *    Close confirm modal dialog.
+     *
+     *    @method hideConfirmModalDialog
+     *    @param {String} confirmCaption Confirm caption (window header caption).
+     *    @param {String} confirmContent Confirm content (window body content).
+     *    @returns {String} Confirm content.
+     */
+    closeDialog() {
+      if (this.confirmModalDialog === undefined) {
+        return;
+      }
+
+      this.set('confirmModalDialogContent', undefined);
+      this.set('confirmModalDialogCategory', undefined);
+      this.confirmModalDialog.modal('hide');
+    }
+
+  },
+
+  makeAssert() {
+    let message = this._getMessageNumber() + 'Assert invocation testing';
+    try {
+      Ember.assert(message, false);
+    } catch (e) {
+      Ember.Logger.error(e);
+      this._router.router.refresh();
+    }
+
+  },
+
+  makeError() {
+    try {
+      eval('error_operator');
```

Comment:
We need generate error message. 
eval('error_operator') generate error message:
error_operator is not defined
with detailed call stack